app/src/components/types/sizing.tsx

```diff
@@ -0,0 +1,7 @@
+export type SizingProps = {
+  /**
+   * The size of the component
+   * @default 'default'
+   */
+  size?: "default" | "compact";
```

Comment:
actually thinking I hate this - thinking we move to S, M, L where L maps to component's default (since it's huge)...